src/core/MOM_barotropic.F90

```diff
@@ -1246,12 +1246,12 @@ subroutine btstep(U_in, V_in, eta_in, dt, bc_accel_u, bc_accel_v, forces, pbce,
         CS%IDatu(I,j) = 0.0
       elseif (integral_BT_cont) then
         CS%IDatu(I,j) = CS%dy_Cu(I,j) / (max(find_duhbt_du(ubt(I,j)*dt, BTCL_u(I,j)), &
-                                             CS%dy_Cu(I,j)*Htot_avg) )
+                                             CS%dy_Cu(I,j)*Htot_avg) ) * GV%Z_to_H
```

Comment:
I can never remember when Fortran is left-associative or right-associative.  In a case like this (and the 3 other similar instances in this file), I would prefer that this be written with the scaling factor out front, as in
    `CS%IDatu(I,j) = GV%Z_to_H * CS%dy_Cu(I,j) / &`
   `       (max(find_duhbt_du(ubt(I,j)*dt, BTCL_u(I,j)), CS%dy_Cu(I,j)*Htot_avg) ) `
or else as
    `CS%IDatu(I,j) = CS%dy_Cu(I,j) / &`
    `      (GV%H_to_Z * max(find_duhbt_du(ubt(I,j)*dt, BTCL_u(I,j)), CS%dy_Cu(I,j)*Htot_avg) ) `

so that it is unambiguous whether the scaling factor is in the numerator or denominator, and there is no need to remember the Fortran associativity rules.